framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/template/TemplateSelector.java

```diff
@@ -0,0 +1,36 @@
+package org.kie.cloud.openshift.template;
+
+import java.util.Arrays;
+
+class TemplateSelector {
+
+    enum Database {
+        POSTGRESQL,
+        MYSQL,
+        GENERAL
+    }
+
+    enum Product {
```

Comment:
@jhrcek One more thing crossed my mind - Is `Product` the proper name for something which has values of `DROOLS` and `JBPM`?

Reply:
Good point changed to Project in last commit